src/services/messages.js

```diff
@@ -15,8 +15,16 @@ export async function sendMessage(email, status = 0) {
 }
 
 export function subscribe(onMessage = (_message) => {}) {
-  // TODO: Subscribe to changes for the `messages` table
-  // and call `onMessage` with the newly added row
+  //   // TODO: Subscribe to changes for the `messages` table
+  //   // and call `onMessage` with the newly added row
+
+  const res = client
+    .from('messages')
+    .on('INSERT', (payload) => {
+      console.log('message received!', payload);
+    })
```

Comment:
We are missing invoking our onMessage function that takes the new property on the payload object. Also, we can get away with omitting the return statement on line 27.